openverse_catalog/dags/providers/smk_workflow.py

```diff
@@ -18,12 +19,12 @@
 
 logger = logging.getLogger(__name__)
 
-DAG_ID = "staten_museum_workflow"
+DAG_ID = "smk_workflow"
```

Comment:
Just a note: Airflow does not handle changing DAG IDs well 😅 We will "lose" history in the Airflow UI for `staten_museum_workflow`, and `smk_workflow` will show up as a new DAG. The DAG runs will still be present in the meta DB, they just won't show up in the UI. That said I think this change is fine 🙂 All of the runs of this DAG have been quick and successful!

Reply:
That's true, it creates a new DAG! Very good call out, I didn't consider that meant we would lose history. But I agree -- luckily this one has much history we'd be interested in keeping 😅